js/login.js

```diff
@@ -4,7 +4,7 @@
     const password = document.getElementById('inputPassword');
 
     form.addEventListener('submit', (e) => {
-        if (username.value !== 'Grupp13') {
+        if (username.value !== 'Grupp7') {
             e.preventDefault(); // Stoppar formuläret från att skicka
```

Comment:
e.preventDefault function can be in front of "if", so it will happen directly when clicking the button